tests/ha/test_data_pod_failure.py

```diff
@@ -2664,24 +2583,24 @@ def test_copy_object_during_pod_shutdown(self):
                                                       bkt_op=False, put_etag=put_etag)
             assert_utils.assert_true(resp[0], f"Failed buckets are: {resp[1]}")
             put_etag = resp[1]
-        LOGGER.info("Step 7: Successfully checked responses from background process.")
+        LOGGER.info("Step 4: Successfully checked responses from background process.")
 
-        LOGGER.info("Step 8: Download the uploaded objects & verify etags")
+        LOGGER.info("Step 5: Download the uploaded objects & verify etags")
         for key, val in bkt_obj_dict.items():
             resp = s3_test_obj.get_object(bucket=key, key=val)
             LOGGER.info("Get object response: %s", resp)
             get_etag = resp[1]["ETag"]
             assert_utils.assert_equal(put_etag, get_etag, "Failed in Etag verification of "
```

Comment:
assert message : 
            assert_utils.assert_equal(put_etag, get_etag, "Failed to match GET-PUT ETAG for "
                                                          f"object {key} of bucket {val}.")
                                                          
 CHECK at all relevant places